DiscordBot.MessageHandlers/AnyoneMentionHandler.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Discord.WebSocket;
+using DiscordBot.Core.Interfaces;
+using Microsoft.Extensions.Logging;
+using System.Text.RegularExpressions;
+
+
+namespace DiscordBot.MessageHandlers
+{
+    public class AnyoneMentionHandler : IMessageReceivedHandler
+    {
+        private static Random random = new Random();
+
+        private const string SELF_PATTERN = @"@anyone";
+        private readonly ILogger<AnyoneMentionHandler> _logger;
+
+        public AnyoneMentionHandler(ILogger<AnyoneMentionHandler> logger)
+        {
+            _logger = logger;
+        }
+
+        private bool IsApplicable(SocketMessage socketMessage)
+            => socketMessage.Channel is SocketTextChannel &&
+            Regex.IsMatch(socketMessage.Content, SELF_PATTERN);
```

Comment:
@Niewidzialny84 This should also **not** be applicable if the author is a bot (`IUser.IsBot`)

Reply:
Resolving - the `MessageReceivedHandlerProvider` is already managing that